cli.js

```diff
@@ -29,10 +29,47 @@ const parser = argly
     '--ui -u': {
       type: 'string',
       description: 'The mocha ui to use. (Defaults to "bdd")'
+    },
+    '--ignoreHTTPSErrors -i': {
+      type: 'boolean',
+      description: 'Whether to ignore HTTPS errors during navigation. Defaults to false.'
+    },
+    '--headless -H': {
+      type: 'boolean',
+      description: 'Whether to run Chromium in headless mode. Defaults to true.'
+    },
+    '--executablePath -e': {
+      type: 'string',
+      description: 'Path to a Chromium executable to run instead of bundled Chromium. If executablePath is a relative path, then it is resolved relative to current working directory.'
+    },
+    '--slowMo -s': {
+      type: 'number',
+      description: 'Slows down Puppeteer operations by the specified amount of milliseconds. Useful so that you can see what is going on.'
+    },
+    '--args -a': {
+      type: 'string[]',
+      description: 'Additional arguments to pass to the Chromium instance. List of Chromium flags can be found at https://peter.sh/experiments/chromium-command-line-switches/.'
+    },
+    '--handleSIGINT -S': {
+      type: 'boolean',
+      description: 'Close chrome process on Ctrl-C. Defaults to true.'
+    },
+    '--timeout -t': {
+      type: 'number',
+      description: 'Maximum time in milliseconds to wait for the Chrome instance to start. Defaults to 30000 (30 seconds). Pass 0 to disable timeout.'
+    },
```

Comment:
Although, we currently don't accept a `timeout` option for `mocha` I could see that being a useful cli option. However the inclusion of that would conflict with this option. Perhaps we should start prefixing these arguments if we plan on supporting both `mocha` and `puppeteer` options from the cli. Usage can get pretty ugly if we do that though.

Since the other arguments are somewhat unique, it might be better to just change this to `puppeteerTimeout` and later on we can add the option for `mochaTimeout`.